components/user/follow/suggestion.service.ts

```diff
@@ -0,0 +1,66 @@
+type SessionStorageSuggestions = {
+	// When user follow suggestions were dismissed.
+	// To not bug with suggestions, we limit them to X per interval.
+	dismissed: number[];
+	[userId: number]: number;
+};
+const SessionStorageSuggestionsKey = 'user-follow-suggestions';
+
+// How many suggestion dismissals we are allowing per interval.
+const MaxSuggestionDismissalsPerDay = 3;
+
+// Whats the interval used to rate limit dismissed suggestions.
+const SuggestionRateLimitInterval = 86400000;
+
+export default abstract class UserFollowSuggestion {
```

Comment:
Don't export default. Only do that for components

Reply:
why, any specific reason?

Reply:
Yeah, it's nicer autocompletion behavior and I don't like the way default imports look. All of our current code is like this too, so it's consistency.